packages/fx-core/src/common/telemetry.ts

```diff
@@ -59,6 +61,11 @@ export function sendTelemetryEvent(
 			properties[TelemetryProperty.Component] = Component.cli;
 		}
 	}
+  
+  const correlationId = inputs.correlationId;
+  if(correlationId)
```

Comment:
Make correlationId a mandatory property, If correlationId is empty, just put empty value for the property. Same for below.